quupe-RN/js/screens/SignIn/SignInContainer.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
```

Comment:
I think your branch has signIn and LogIn, both screens. Please fix it and use what everybody is using.